package.json

```
@@ -61,7 +62,8 @@
   },
   "files": [
     "icons/*.svg",
-    "bootstrap-icons.svg"
+    "bootstrap-icons.svg",
+    "fonts"
```

Better be more explicit here like this:

```suggestion
    "fonts/*.{eot,ttf,woff,woff2}"
```

---

I want the whole fonts directory for now. Too weird?

---

Yup. Because unneeded files might be included in the npm package, even files not tracked by git.

---

Actually, now that I see the folder contents, ignore my comment. It's just too many filetypes to specify.